tool/options.yml

```diff
@@ -98,6 +99,10 @@ BACKTRACES_OMIT_UNUSED: [backtraces.omit_unused, boolean, true, Omit backtraces
 BASICOPS_INLINE: [basic_ops.inline, boolean, true, Inline basic operations (like Fixnum operators) in the AST without a call]
 
 GRAAL_WARNING_UNLESS: [graal.warn_unless, boolean, true, Warn unless the JVM has the Graal compiler]
+GRAAL_PRESENT: [graal.present, boolean, true, "On graal?"]
+
+RUBY_COPYRIGHT: [ruby_copyright, string, '', Copyright]
+RUBY_DESCRIPTION: [ruby_description, string, '', Ruby description]
```

Comment:
What are these options? I thought they were about printing the copyright, but why are the strings themselves options?